packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_navigation_delegate.dart

```diff
@@ -86,6 +89,16 @@ abstract class PlatformNavigationDelegate extends PlatformInterface {
         'setOnPageFinished is not implemented on the current platform.');
   }
 
+  /// Invoked when an error has occurred during loading
+  ///
+  /// See [PlatformWebViewController.setPlatformNavigationDelegate].
+  Future<void> setOnPageError(
```

Comment:
It seems that this method is looking to specifically handle http error codes. I think `setOnHttpError` would be more appropriate  than `setOnPageError` which seems too broad for what the implementations are actually doing.